ktor-client/ktor-client-okhttp/build.gradle.kts

```diff
@@ -3,7 +3,7 @@ apply<test.server.TestServerPlugin>()
 kotlin.sourceSets {
     jvmMain {
         dependencies {
-            api(project(":ktor-client:ktor-client-core"))
+            implementation(project(":ktor-client:ktor-client-plugins:ktor-client-sse"))
```

Comment:
it's a bit strange that client engine depends on the plugin. should it be split?